scripts/inboxtable.js

```diff
@@ -9,13 +9,6 @@
         $.fn.dataTableExt.oStdClasses.sStripeOdd = "row r0";
         $.fn.dataTableExt.oStdClasses.sStripeEven = "row r1";
 
-        // Decide table ordering based on langconfig setting
-        if (date_format ==  "%d/%m/%y, %H:%M") {
```

Comment:
Well pointed out, I have removed these additional redundancies. The sort is now performed on unix time so the date format is no longer used.